scripts/genetic_backtester/darwin.js

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env node
 
 /* Zenbot 4 Genetic Backtester
- * Clifford Roche <clifford.roche@gmail.com>
- * 07/01/2017
- *
- * Example: ./darwin.js --selector="bitfinex.ETH-USD" --days="10" --currency_capital="5000" --use_strategies="all | macd,trend_ema,etc" --population="101" --population_data="simulations/generation_data_NUMBERS_gen_X.json"
- */
+* Clifford Roche <clifford.roche@gmail.com>
```

Comment:
I'd argue it looks better with a space in front of it, might be personal preference